### ISSUE TITLE
Fix docs for auto_lr_find

### ISSUE THREAD

**Issue by edenlightning** [PyTorchLightning/pytorch-lightning#3799]
This is the correct way to run

```
trainer = pl.Trainer(auto_lr_find=True)

lr_finder = trainer.tuner.lr_find(model) # Run learning rate finder

fig = lr_finder.plot(suggest=True) # Plot
fig.show()

model.hparams.learning_rate = lr_finder.suggestion()
```

**Comment by Borda** [comment]
@edenlightning mind share where is the wring docs?

**Comment by SkafteNicki** [comment]
Closing this as it was solved by #3883

**Comment by thepurpleowl** [comment]
For future readers, it they stumble upon this issue in future ,
refer the following documentation - [lr_finder](https://pytorch-lightning.readthedocs.io/en/latest/advanced/lr_finder.html)
